php/libraries/LorisForm.class.inc

```diff
@@ -49,6 +49,8 @@ class LorisForm
         'email',
         'maxlength',
     );
+    public $validOperators;
```

Comment:
I don't understand this one. It looks like it's never set in the code and I don't see why it's public.

Reply:
It's used by `function validateRequireIf`... sort of. If you take a look at that function it looks like it needs a bit of refactoring. Should I do that in this PR or no?

Reply:
yeah, but it's only used in this context:

```
            throw new LorisException(
                "operator is not a valid operator from the set "
                . join(', ', $this->validOperators)
            );
```

and it's never set.. when I grep the code for 'validOperators' it's also never set externally even though it's public. I guess you're right that it needs refactoring and the reference to it seems to be a bug, but fixing it properly is out of the scope of this PR.

Reply:
Do you object to adding this variable so that we can satisfy phpstan for now and leave refactoring for the future?

Reply:
No objections, I think it's fine to satisfy phpstan.